apps/api/src/app.module.ts

```diff
@@ -14,8 +17,9 @@ import { InternModule } from './intern/intern.module';
       exclude: ['/api/(.*)'],
     }),
     InternModule,
+    AuthModule,
   ],
-  controllers: [AppController],
-  providers: [AppService, PrismaService],
+  controllers: [AppController, AuthController],
+  providers: [AppService, PrismaService, AuthService],
```

Comment:
baca mi error ako je `AuthService` includean ovdje
```suggestion
  providers: [AppService, PrismaService],
```